codex/node.nim

```diff
@@ -387,6 +388,13 @@ proc start*(node: CodexNodeRef) {.async.} =
       error "Unable to start client contract interactions: ", error=error.msg
       node.contracts.client = ClientInteractions.none
 
+  if validatorContracts =? node.contracts.validator:
+    try:
+      await validatorContracts.start()
+    except CatchableError as error:
```

Comment:
Are there specific error types that we could catch that may give better error info in the log and provide a better debugging story?

Reply:
See https://github.com/status-im/nim-codex/pull/387#discussion_r1165421772